### PR TITLE
adding t-test to pre-deployed models

### PR DESCRIPTION
**Pull request by sbabayan**

### CODE REVIEW

File: tests/integration/test_deploy_model_ssl_off_auth_off.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def test_deploy_ssl_off_auth_off(self):
         self.assertEqual(200, SentimentAnalysis_request.status)
         SentimentAnalysis_request.read()
 
-        conn.request("GET", "/endpoints/tTest")
-        tTest_request = conn.getresponse()
-        self.assertEqual(200, tTest_request.status)
-        tTest_request.read()
+        conn.request("GET", "/endpoints/ttest")
```
Comment (0golovatyi) [inline tests/integration/test_deploy_model_ssl_off_auth_off.py]: this code is repeated for 3 times now. Can we keep configurations separately (in a dict?) and just run the same code for every configuration?

File: docs/tabpy-tools.md [hunk 2]
```diff
@@ -265,6 +265,48 @@ tabpy.query('Sentiment Analysis', _arg1, library='textblob')[‘response’]
 
 ```
 
+### T-Test
```
Comment (0golovatyi) [inline docs/tabpy-tools.md]: Run `markdown-toc` to update TOC for the document.

File: models/scripts/tTest.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        return p_value
+    # arg1 is a sample and arg2 is a sample
+    else:
+        test_stat, p_value = stats.ttest_ind(_arg1, _arg2, equal_var=False)
+        return p_value
+
+
+if __name__ == '__main__':
+    # running from setup.py
+    if len(sys.argv) > 1:
```
Comment (0golovatyi) [inline models/scripts/tTest.py]: Code from L38 to L58 is now in multiple files with the only difference in function name and description. We should move it to utilities and reuse it.

File: docs/tabpy-tools.md (hunk 2 above)
Comment (sbabayan) [reply docs/tabpy-tools.md]: fixed

File: tests/integration/test_deploy_model_ssl_off_auth_off.py (hunk 1 above)
Comment (sbabayan) [reply tests/integration/test_deploy_model_ssl_off_auth_off.py]: fixed

**Comment by 0golovatyi** [comment]
There are build failures

File: models/scripts/tTest.py (hunk 3 above)
Comment (sbabayan) [reply models/scripts/tTest.py]: fixed

**Pull request merged**